src/Components/InputField.jsx

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import '../Themes/inputfield.css';
```

Comment:
Fixes the build  error.
```suggestion
import '../Themes/InputField.css';
```